app/models/vote.rb

```diff
@@ -7,6 +7,8 @@ class Vote < ApplicationRecord
   scope :article_top_10_month, -> { where("voteable_type = 'Article' AND created_at >= ?", DateTime.now - 1.month).group(:voteable_id).sum(:value) }
   scope :article_top_10_day, -> { where("voteable_type = 'Article' AND created_at >= ?", DateTime.now - 1.day).group(:voteable_id).sum(:value) }
   
+  scope :user_articles_voted, lambda{|user| where(user: user, voteable_type: "Article").order(updated_at: :desc).pluck(:voteable_id) }
```

Comment:
Above in the other scope, you use the `-> { expr }` shorthand syntax, but you use `lambda { expr }` long syntax here. Normally scopes use the arrow.

Reply:
Also, this scope is not chainable since you `pluck(:id)` at the end. What do you think of taking that part off? And pluck the IDs elsewhere.

Reply:
I didn't know you could still use the arrow and still pass in an argument to the scope. 
And yep, I'll go ahead and use the pluck somewhere else!